src/winetricks

```diff
@@ -11240,18 +11240,16 @@ load_fontxplorer()
 #----------------------------------------------------------------
 
 w_metadata foobar2000 apps \
-    title="foobar2000 v1.3.14" \
+    title="foobar2000 v1.3.17" \
     publisher="Peter Pawlowski" \
-    year="2016" \
+    year="2018" \
     media="manual_download" \
-    file1="foobar2000_v1.3.14.exe" \
+    file1="foobar2000_v1.3.17.exe" \
     installed_exe1="$W_PROGRAMS_X86_WIN/foobar2000/foobar2000.exe"
 
 load_foobar2000()
 {
-    # 2016/12/21: 1.3.14 - 72d024d258c2f3b6cea62dc47fb613848202e7f33f2331f6b2e0a8e61daffcb6
-
```

Comment:
Don't remove that. Please add the new shasum as well, with the date you updated it, as a comment.